### ISSUE TITLE
Bounds for base are off

### ISSUE THREAD

**Issue by lehins** [NorfairKing/validity#77]
The lower bound on genvalidity for base should be set to to `base >= 4.10`
```
genvalidity       > /tmp/stack-3feadd8e35b44dfe/genvalidity-0.10.0.1/src/Data/GenValidity/Utils.hs:274:22: error:
genvalidity       >     Variable not in scope: castWord32ToFloat :: w0 -> Float
genvalidity       >     |                         
genvalidity       > 274 | genFloat = genFloatX castWord32ToFloat
genvalidity       >     |                      ^^^^^^^^^^^^^^^^^
genvalidity       >                               
genvalidity       > /tmp/stack-3feadd8e35b44dfe/genvalidity-0.10.0.1/src/Data/GenValidity/Utils.hs:278:23: error:
genvalidity       >     Variable not in scope: castWord64ToDouble :: w1 -> Double
genvalidity       >     |                         
genvalidity       > 278 | genDouble = genFloatX castWord64ToDouble
genvalidity       >     |                       ^^^^^^^^^^^^^^^^^^
genvalidity       >                               
```

**Comment by NorfairKing** [comment]
@lehins Thanks for figuring that out. I'll add the lower bound now.

This stuff annoys me to no end. How am I supposed to figure this out?!

**Comment by NorfairKing** [comment]
Fixed in `0.10.0.2`.

**Comment by lehins** [comment]
Extensive CI. Try compiling your package with as many ghc versions as possible and see what breaks. That's exactly how I figured this ticket out.

You can make a revision Hackage, there is no need to make new releases just for updating the bounds.
